### ISSUE TITLE
TypeError: default value of Tuple was None Error

### ISSUE THREAD

**Issue by gerard0315** [bernhard-42/cad-viewer-widget#4]
Hi, big thanks to your efforts here.

I tried installethe package in both Python 3.5 and Python 3.8 but encountered the error when I run `import cad_viewer_widget`

```
TypeError                                 Traceback (most recent call last)
<ipython-input-8-87939327274d> in <module>
----> 1 import cad_viewer_widget
      2 import numpy as np
      3 import json
      4 import time
      5 import ipywidgets as widgets

/opt/conda/lib/python3.8/site-packages/cad_viewer_widget/__init__.py in <module>
      4 from IPython.display import display, HTML
      5 
----> 6 from .widget import AnimationTrack, CadViewer
      7 from .sidecar import Sidecar
      8 

/opt/conda/lib/python3.8/site-packages/cad_viewer_widget/widget.py in <module>
    104 
    105 @widgets.register
--> 106 class CadViewerWidget(widgets.Output):  # pylint: disable-msg=too-many-instance-attributes
    107     """The CAD Viewer widget."""
    108 

/opt/conda/lib/python3.8/site-packages/cad_viewer_widget/widget.py in CadViewerWidget()
    219     "bool: Whether to show colored clipping planes (True) or not (False)"
    220 
--> 221     clip_normal_0 = Tuple(Float(), Float(), Float(), allow_none=True, default_value=None).tag(sync=True)
    222     "tuple: Normal of clipping plane 1 as a 3-dim tuple of float (x,y,z)"
    223 

/opt/conda/lib/python3.8/site-packages/traitlets/traitlets.py in __init__(self, *traits, **kwargs)
   2414             args = (default_value,)
   2415         else:
-> 2416             raise TypeError('default value of %s was %s' %(self.__class__.__name__, default_value))
   2417 
   2418         self._traits = []

TypeError: default value of Tuple was None

```

is there any particular requirements for Python version here or am I just using the package incorrectly

**Comment by bernhard-42** [comment]
Thanks for giving it a try. 

Just verified on a Mac with python 3.8 in a fresh environment (not sure why I suddenly need pyparsing ...)

```python
mamba create -n cvw python=3.8
mamba activate cvw
pip install cad_viewer_widget pyparsing
```

```bash
$ ipython
Python 3.8.15 | packaged by conda-forge | (default, Nov 22 2022, 08:49:06)
Type 'copyright', 'credits' or 'license' for more information
IPython 8.7.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import cad_viewer_widget

In [2]:
```

Here are some libs with their version in my environment:

```shell
$ pip list | egrep '(traitlets|ipywidgets|ipython)'

ipython              8.7.0
ipython-genutils     0.2.0
ipywidgets           7.7.2
traitlets            5.8.0
```

Which versions do you have?

**Comment by gerard0315** [comment]
```
ipython                       7.17.0
ipython-genutils              0.2.0
ipywidgets                    7.7.2
traitlets                     4.3.3
```
i'll try upgrading trailets and ipywidgets to see if it works

**Comment by gerard0315** [comment]
turns out  i have to upgrade traitlets and ipywidgets to make it work, thanks

**Comment by bernhard-42** [comment]
fyi, I've published 1.4.1 that adds `pyparsing` to the dependencies and tags `traitlets` to `~=5.8`
